packages/integration-tests/tests/transfer_eth.ts

```diff
@@ -343,4 +347,167 @@ describe('Transferring ETH', async function () {
       ).to.be.true;
     });
   });
+
+  describe('ETH Withdrawls based on rate limit updates', async () => {
+    let NUM_ETH = '9';
+    let fuelETHSender: FuelWallet;
+    let ethereumETHReceiver: Signer;
+    let ethereumETHReceiverAddress: string;
+    let withdrawMessageProof: MessageProof;
+    let rateLimitDuration: bigint;
+
+    before(async () => {
+      fuelETHSender = env.fuel.signers[1];
+      ethereumETHReceiver = env.eth.signers[1];
+      ethereumETHReceiverAddress = await ethereumETHReceiver.getAddress();
+    });
+
+    it('Checks rate limit params after relaying', async () => {
+      withdrawMessageProof = await generateWithdrawalMessageProof(
+        fuelETHSender,
+        ethereumETHReceiverAddress,
+        NUM_ETH
+      );
+
+      await relayMessage(env, withdrawMessageProof);
+
+      const totalETHWithdrawn = new BN((9.002e18).toString());
```

Comment:
Magic number here: why 9.002e18? 

Also, no need to use Fuel 's BN object here, we can use javascript bigints, eg `9n * (10n ** 18n)`  or ethers utility: `parseEther(9.002)`